index.js

```diff
@@ -817,6 +817,10 @@ class Client {
     return this.initialUpdateConfig;
   }
 
+  getAmpConfig() {
+    return this.request("/api/amp/config")
```

Comment:
Did requested change.
Was calling `/api/amp/config` because it was mentioned in `https://github.com/quintype/itsman/issues/4982`.  But Spoke with Gopal and now changing this to `api/v1/amp/config`